src/utils.js

```diff
@@ -0,0 +1,204 @@
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+function restoreOriginalIcons() {
+  const iconDiv = document.querySelectorAll('.icon');
+  iconDiv.forEach((icon) => {
+    icon.innerHTML = '&#8942';
+    icon.style.cursor = 'all-scroll';
+  });
+}
+
+function restoreBackgroundWhite() {
+  const labelTasks = document.querySelectorAll('li');
+  labelTasks.forEach((label) => {
+    label.style['background-color'] = '#fff';
+  });
+}
+
+function scrapeAndSave() {
+  const tasksArray = [];
+  const tasks = document.querySelectorAll('.task');
+  tasks.forEach((label, index) => {
+    const parentTaskCont = label.parentNode;
+    const checkboxElem = parentTaskCont.querySelector('.checkBoxesTasks');
+    const task = new Task(label.innerHTML, checkboxElem.checked, index + 1);
+    tasksArray.push(task);
+  });
+  localStorage.removeItem('tasksList');
+  localStorage.setItem('tasksList', JSON.stringify(tasksArray));
+}
+
+function setValueTextArea(textArea, labelTask) {
```

Comment:
- [ ] Great job on adding the update and remove all completed features however there is a little problem that I deticted which needs to be fixed. According to the requirements the updated array of items should be stored in local storage, so the user gets the correct list values after the page reloads. However when I reload the page the fetched array values did not have the updated value for completed. Please make sure to reload the page to test the array is stored. See screencast for more details:

https://github.com/donice/TDL-Microverse/assets/44798044/c1e01ee3-c486-401f-abf0-3dea9648ad70